modules/cfe_assert/src/cfe_assert_runner.c

```diff
@@ -69,6 +69,26 @@ static CFE_EVS_BinFilter_t CFE_TR_EventFilters[] = {
     {UTASSERT_CASETYPE_DEBUG, CFE_EVS_NO_FILTER},
 };
 
+bool CFE_UtAssert_StatusCheck(CFE_Status_t Status, bool ExpectSuccess, UtAssert_CaseType_t CaseType, const char *File,
+                              uint32 Line, const char *Text)
+{
+    bool Result = (Status >= CFE_SUCCESS);
+    char Disp;
+
+    if (ExpectSuccess)
+    {
+        Disp = '=';
+    }
+    else
+    {
+        /* expecting non-success; result should be inverted */
+        Result = !Result;
+        Disp   = '!';
+    }
+
+    return UtAssertEx(Result, CaseType, File, Line, "%s (0x%lx) %c= CFE_SUCCESS", Text, (unsigned long)Status, Disp);
```

Comment:
Log message should match the test actually performed.

Reply:
OK, updated again to tweak the assert string output.  I also temporarily hacked a couple test cases to use this macro and get a sample output.  In my latest push (4f9d3df9) it looks like this in the final log:

```
[ PASS] 01.001 es_application_control_test.c:42 - CFE_ES_GetAppID(&TestAppId) (0x0) is OK
[ PASS] 01.002 es_application_control_test.c:44 - CFE_ES_RestartApp(CFE_ES_APPID_UNDEFINED) (0xffffffffc4000001) is ERROR
```